### ISSUE TITLE
"Cannot convert path" when a failed spec contains `s/keys*`

### ISSUE THREAD

**Issue by j-cr** [bhb/expound#112]
Quick example:

```clojure
(ns user.test
  (:require [clojure.spec.alpha :as s] 
            [expound.alpha :as expound]))

(set! s/*explain-out* expound/printer)

(s/def ::a #{1})
(s/def ::b #{2})

(s/explain (s/keys* :opt-un [::a ::b]) [:a "foo"])
;; =>
;; ExceptionInfo Cannot convert path. This can be caused by using conformers to transform values, which is not supported in Expound  clojure.core/ex-info
```

As far as I understand, this is similar to #102 and would be resolved (kind of) by #78, but since there are no issues mentioning `s/keys*` specifically I decided to create this issue anyway in case someone else runs into this.

By the way, thanks for creating this library, it's a must-have!

**Comment by bhb** [comment]
Thanks for reporting this! Looks like a bug to me.

**Comment by bhb** [comment]
@j-cr I've got a fix for this issue in 0.7.2-SNAPSHOT. If you have time to try it out, please let me know what you think!

**Comment by bhb** [comment]
Fixed in 0.7.2-SNAPSHOT (to be released soon)
